bin/localicious-render

```diff
@@ -1,59 +1,86 @@
 #!/usr/bin/env node
 const path = require("path");
 const program = require("commander");
+const keywords = require("../src/model/keywords");
 const validateAndRender = require("../src/actions/validateAndRender");
 
 const commaSeparatedList = value => {
   return value.split(",");
 };
 
-program
-  .option("-l --languages <languages>", "output languages", commaSeparatedList)
-  .option("-p --platforms <platforms>", "target platforms", commaSeparatedList)
-  .parse(process.argv);
+const hasValidLanguages = languages => {
+  if (languages === undefined || languages.length == 0) {
+    console.error(`❌ This command requires one or more output languages.`);
+    return false;
+  }
+  return true;
+};
 
-if (program.args.length < 2) {
-  console.log(`Usage: localicious render <localicipe> <output_path> -l <languages> -p <platforms>`);
-  process.exit(1);
-}
+const hasValidCollections = collections => {
+  if (collections === undefined || collections.length == 0) {
+    console.error(`❌ This command requires one or more string collections.`);
+    return false;
+  }
+  return true;
+};
 
-const localicipePath = path.resolve(process.cwd(), program.args[0]);
-const outputPath = path.resolve(process.cwd(), program.args[1]);
+const hasValidOutputTypes = outputTypes => {
+  const supportedOutputTypes = Object.keys(keywords.outputType).map(s => s.toLowerCase());
 
-if (program.languages === undefined || program.languages.length == 0) {
-  console.error(`This command requires one or more output languages.`);
-  process.exit(1);
-}
+  if (outputTypes === undefined || outputTypes.length == 0) {
+    console.error(
+      `❌ This command requires one or more output types. Supported output types: '${supportedOutputTypes.join(
+        " | "
+      )}'.`
+    );
+    return false;
+  }
 
-const supportedPlatforms = ["ios", "android"];
-if (program.platforms === undefined || program.platforms.length == 0) {
-  console.error(
-    `This command requires one or more target platforms. Supported target platforms: '${supportedPlatforms.join(
-      " | "
-    )}'.`
-  );
-  process.exit(1);
-}
+  const invalidOutputTypes = outputTypes.filter(option => !supportedOutputTypes.includes(option));
 
-const invalidPlatforms = program.platforms.filter(option => {
-  return !supportedPlatforms.includes(option);
-});
+  if (invalidOutputTypes.length != 0) {
+    console.error(
+      `❌ Received invalid output types: '${invalidOutputTypes.join(
+        " | "
+      )}'. Supported output types: '${supportedOutputTypes.join(" | ")}'.`
+    );
+    return false;
+  }
+  return true;
+};
 
-if (invalidPlatforms.length != 0) {
-  console.error(
-    `Received invalid target platforms: '${invalidPlatforms.join(
-      " | "
-    )}'. Supported target platforms: '${supportedPlatforms.join(" | ")}'.`
+program
+  .option("-l --languages <languages>", "output languages", commaSeparatedList)
+  .option("-ot --outputTypes <outputTypes>", "output types", commaSeparatedList)
```

Comment:
```suggestion
  .option("-ot --outputTypes <output_types>", "output types", commaSeparatedList)
```

To be [consistent](https://github.com/PicnicSupermarket/localicious/blob/master/bin/localicious#L9).